addons/mrp_account/wizard/mrp_wip_accounting.py

```diff
@@ -0,0 +1,150 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime, time
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models, _, api, Command
+from odoo.exceptions import UserError
+from odoo.tools import format_list
+
+
+class MrpWipAccountingLine(models.TransientModel):
+    _name = 'mrp.account.wip.accounting.line'
+    _description = 'Account move line to be created when posting WIP account move'
+
+    account_id = fields.Many2one('account.account', "Account")
+    label = fields.Char("Label")
+    debit = fields.Monetary("Debit", compute='_compute_debit', store=True, readonly=False)
+    credit = fields.Monetary("Credit", compute='_compute_credit', store=True, readonly=False)
+    currency_id = fields.Many2one('res.currency', "Currency", default=lambda self: self.env.company.currency_id)
+    wip_accounting_id = fields.Many2one('mrp.account.wip.accounting', "WIP accounting wizard")
+
+    _sql_constraints = [
+        ('check_debit_credit', 'CHECK ( debit = 0 OR credit = 0 )',
+         'A single line cannot be both credit and debit.')
+    ]
+
+    @api.depends('credit')
+    def _compute_debit(self):
+        for record in self:
+            if not record.currency_id.is_zero(record.credit):
+                record.debit = 0
+
+    @api.depends('debit')
+    def _compute_credit(self):
+        for record in self:
+            if not record.currency_id.is_zero(record.debit):
+                record.credit = 0
+
+
+class MrpWipAccounting(models.TransientModel):
+    _name = 'mrp.account.wip.accounting'
+    _description = 'Wizard to post Manufacturing WIP account move'
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super().default_get(fields_list)
+        productions = self.env['mrp.production'].browse(self.env.context.get('active_ids'))
+        # ignore selected MOs that aren't a WIP
+        productions = productions.filtered(lambda mo: mo.state in ['progress', 'to_close', 'confirmed'])
+        if 'journal_id' in fields_list:
+            default = self.env['ir.property']._get_default_property('property_stock_journal', 'product.category')[1]
```

Comment:
I checked with the PR's author and it sounds like since we're only using `env['ir.property']._get_default_property()`, it should be fine as is. Worst case scenario, we'll change it if it seems like something broke :sweat_smile: